app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -328,12 +328,15 @@ ensogl::define_endpoints_2! {
         /// Press event. Emitted when user clicks on non-active part of the node, like its
         /// background. In edit mode, the whole node area is considered non-active.
         background_press         (),
-        /// Emitted when node expression is modified as a whole. Does not include partial changes on
-        /// individual spans, which are emitted via `expression_span` output.
-        expression               (ImString),
-        /// Emitted when node expression is edited in context of specific span. Does not include
-        /// changes to the expression as a whole, which are emitted via `expression` output.
-        expression_span          (span_tree::Crumbs, ImString),
+        /// This event occurs when the user modifies an expression, either by typing directly or
+        /// using a widget. It includes information about the specific part of the expression that
+        /// was changed and where it fits within the larger expression.
+        ///
+        /// Note: Node component is not able to perform the actual modification of the expression,
+        /// as that requires rebuilding the span-tree, which in turn requires access to the
+        /// execution context. It is the responsibility of the parent component to apply the changes
```

Comment:
what is "parent component"? Would you mind adding here a super short explanation of this term, please?